src/main/java/seedu/condonery/logic/parser/ParserUtil.java

```diff
@@ -206,7 +206,8 @@ public static PropertyTypeEnum parsePropertyType(String propertyType) throws Par
      */
     public static PropertyStatusEnum parsePropertyStatus(String propertyStatus) throws ParseException {
         requireNonNull(propertyStatus);
-        return PropertyStatusEnum.valueOf(propertyStatus);
+        return PropertyStatusEnum.valueOf(propertyStatus) == null
```

Comment:
I think this conditional doesn't make sense. valueOf will never return a null value but instead throw IllegalArgumentException, which you will need to catch and throw a ParseException, as stated in the method signature.

You can look at the parsePropertyType function as reference for how I previously catch the error. For example, the command `add -p a/Duxton n/Duxton p/10000 h/hdb s/INVALID VALUE` will cause the application to throw an Exception.